Pipeline now fails due to update to blazegraph-runner

From https://github.com/geneontology/pipeline/pull/254

Once this PR was taken, pipeline runs on `master` now fail with errors like:

```
20:32:01  JAVA_OPTS=-Xmx128G blazegraph-runner --journal=target/blazegraph.jnl --properties=conf/blazegraph.properties reason --source-graphs-query=' PREFIX dc: <http://purl.org/dc/elements/1.1/> SELECT ?source_graph WHERE { GRAPH ?source_graph { ?source_graph dc:title ?title . } }' --ontology=http://purl.obolibrary.org/obo/go/extensions/go-graphstore.owl --append-graph-name="_inferred"
20:32:02  log4j:WARN No appenders could be found for logger (com.bigdata.config.Configuration).
20:32:02  log4j:WARN Please initialize the log4j system properly.
20:32:02  log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
20:32:02  ERROR: com.bigdata.util.config.LogUtil : Could not initialize Log4J logging utility.
20:32:02  Set system property '-Dlog4j.configuration=file:bigdata/src/resources/logging/log4j.properties'
20:32:02    and / or 
20:32:02  Set system property '-Dlog4j.primary.configuration=file:<installDir>/bigdata/src/resources/logging/log4j.properties'
20:32:02  
...
20:33:38  java.lang.NoClassDefFoundError: scribe/Level$Debug$
20:33:38  	at org.renci.blazegraph.Reason$ArachneReasoner.$anonfun$ontRules$1(Reason.scala:199)
20:33:38  	at scala.collection.StrictOptimizedIterableOps.flatMap(StrictOptimizedIterableOps.scala:117)
20:33:38  	at scala.collection.StrictOptimizedIterableOps.flatMap$(StrictOptimizedIterableOps.scala:104)
20:33:38  	at scala.collection.immutable.Set$Set1.flatMap(Set.scala:156)
20:33:38  	at org.renci.blazegraph.Reason$ArachneReasoner.<init>(Reason.scala:199)
...
20:33:38  	at scala.App.main$(App.scala:71)
20:33:38  	at org.renci.blazegraph.Main$.main(Main.scala:5)
20:33:38  	at org.renci.blazegraph.Main.main(Main.scala)
20:33:38  Caused by: java.lang.ClassNotFoundException: scribe.Level$Debug$
20:33:38  	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
20:33:38  	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
20:33:38  	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
20:33:38  	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
20:33:38  	... 26 more
20:33:38  Makefile:201: recipe for target 'target/blazegraph.jnl' failed
20:33:38  make: *** [target/blazegraph.jnl] Error 1
```
https://build.geneontology.org/job/geneontology/job/pipeline/job/master/648/console

Not a showstopper as we could revert to the previous version.

Tagging @balhoff

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -326,13 +326,13 @@ pipeline {
 		    },
 		    "Ready blazegraph-runner": {
 			dir('./blazegraph-runner') {
-			    sh 'wget -N https://github.com/balhoff/blazegraph-runner/releases/download/v1.6.5/blazegraph-runner-1.6.5.tgz'
-			    sh 'tar -xvf blazegraph-runner-1.6.5.tgz'
+			    sh 'wget -N https://github.com/balhoff/blazegraph-runner/releases/download/v1.4/blazegraph-runner-1.4.tgz'
+			    sh 'tar -xvf blazegraph-runner-1.4.tgz'
 			    withCredentials([file(credentialsId: 'skyhook-private-key', variable: 'SKYHOOK_IDENTITY')]) {
 				// Attempt to rsync bin into bin/.
-				sh 'rsync -avz -e "ssh -o StrictHostKeyChecking=no -o IdentitiesOnly=true -o IdentityFile=$SKYHOOK_IDENTITY" blazegraph-runner-1.6.5/bin/* skyhook@skyhook.berkeleybop.org:/home/skyhook/$BRANCH_NAME/bin/'
+				sh 'rsync -avz -e "ssh -o StrictHostKeyChecking=no -o IdentitiesOnly=true -o IdentityFile=$SKYHOOK_IDENTITY" blazegraph-runner-1.4/bin/* skyhook@skyhook.berkeleybop.org:/home/skyhook/$BRANCH_NAME/bin/'
 				// Attempt to rsync libs into lib/.
-				sh 'rsync -avz -e "ssh -o StrictHostKeyChecking=no -o IdentitiesOnly=true -o IdentityFile=$SKYHOOK_IDENTITY" blazegraph-runner-1.6.5/lib/* skyhook@skyhook.berkeleybop.org:/home/skyhook/$BRANCH_NAME/lib/'
+				sh 'rsync -avz -e "ssh -o StrictHostKeyChecking=no -o IdentitiesOnly=true -o IdentityFile=$SKYHOOK_IDENTITY" blazegraph-runner-1.4/lib/* skyhook@skyhook.berkeleybop.org:/home/skyhook/$BRANCH_NAME/lib/'
 			    }
 			}
 		    },

